.eslintignore

```diff
@@ -1,4 +1,3 @@
 *.min.*
 src/lib/markbind/src/lib/markdown-it/*
-
-!.eslintrc.js
+.eslintrc.js
```

Comment:
Linting this file is intentional. It is ignored by default, so this line would be redundant anyway.

Reply:
Without this line `npm run lint` seems to be picking up `.eslintrc.js` on my machine, not sure why it doesn't seem to be consistent.